mmv1/templates/terraform/examples/alloydb_instance_basic.tf.erb

```diff
@@ -0,0 +1,27 @@
+resource "google_alloydb_instance" "<%= ctx[:primary_resource_id] %>" {
+  provider      = google-beta
+  cluster_id    = google_alloydb_cluster.<%= ctx[:primary_resource_id] %>.cluster_id
+  instance_id   = "<%= ctx[:vars]['alloydb_instance_name'] %>"
+  location      = "us-central1"
+  instance_type = "PRIMARY"
+}
+
+resource "google_alloydb_cluster" "<%= ctx[:primary_resource_id] %>" {
+  provider   = google-beta
+  cluster_id = "<%= ctx[:vars]['alloydb_cluster_name'] %>"
+  location   = "us-central1"
+  network    = "projects/${data.google_project.project.number}/global/networks/${data.google_compute_network.my_network.name}"
```

Comment:
This test is currently failing with the following error:

```
Invalid resource state for \"projects/<number>/global/networks/default\": service networking config validation failed NETWORK_PEERING_DELETED - no peering found on network, see https://cloud.google.com/alloydb/docs/configure-connectivity for details
```

Would it be possible to configure a network explicitly instead of using the default?

Reply:
done